docs/readmes/lte/GTPU_Extension_Header.md

```diff
@@ -0,0 +1,149 @@
+---
+id: GTPU_Ext_Header
+title: GTP Extensions
+hide_title: true
+---
+# GTP Extensions
+
+## Overview
+
+Magma Gateway uses the Linux networking stack and OVS to program the packet pipeline on the gateway. The GTP-U protocol entity provides packet transmission and reception services to user plane entities in the RNC, eNodeB, SGW, PGW and TWAN. The GTP-U protocol entity receives traffic from a number of

```

Comment:
We need to mention that this is patched OVS that adds support for kernel based GTP tunnel implementation.

Reply:
Yes,Added